style.css

```diff
@@ -1,7 +1,7 @@
 html { background:#000; }
 body { position:relative; color:#fff; font:normal 1em FreeSans,Helvetica,sans-serif; }
 
-#wrapper { width:480px; padding:50px; margin:0; }
+#wrapper { font-size: 1.5em; width:480px; padding:50px; margin:0; }
```

Comment:
The commit message doesn't say why this change was made, and what it results in exactly.

Reply:
I had set the font-size on body to 1em in the previous commit, this adds back the 1.5em font on a wrapper so the interface still has the same font size

Reply:
Got it, thx.